roles/deploy_ioc/tasks/process-env.yml

```diff
@@ -1,9 +1,17 @@
 ---
 
+- name: Show env var to process
+  ansible.builtin.debug:
+    msg: "Processing {{ item }} with value {{ deploy_ioc_merged_env[item] }}"
+
 - name: Identify macros in envvar value
   ansible.builtin.set_fact:
     deploy_ioc_macros_in_var: "{{ deploy_ioc_merged_env[item] |
-                        regex_findall('\\$\\((.[^\\)]+)\\)') }}"
+                        regex_findall('\\$\\([A-Za-z_][A-Za-z0-9_]*\\)') }}"
```

Comment:
This change broke deployment. You don't capture the correct group anymore leading to it matching the entire regex.

For example,
```
IOC_DIR: /epics/iocs
TOP: $(IOC_DIR)/mc01
```

Before, when processing `TOP` this regex would capture `IOC_DIR`.
After this change, when processing `TOP` this regex captures `$(IOC_DIR)` which fails later validation